src/rpc/util.cpp

```diff
@@ -75,6 +75,13 @@ CAmount AmountFromValue(const UniValue& value, int decimals)
     return amount;
 }
 
+CFeeRate ParseFeeRate(const UniValue& json)
```

Comment:
nit: Would have named it `ParseReasonableFeeRate` or `ParseLimitedFeeRate` to make clear it's doing more than just parsing.

Reply:
Alternatively could have made the limit a parameter, because that also makes it clearer.